src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -108,12 +108,20 @@ LtiConfigForm.propTypes = {
     consumerKey: PropTypes.string.isRequired,
     consumerSecret: PropTypes.string.isRequired,
     launchUrl: PropTypes.string.isRequired,
-  }).isRequired,
```

Comment:
why `isRequired` is removed

Reply:
This was removed because we use the `defaultProps` below to set a default value for appConfig if one wasn't provided.  This can happen if we're configuring a different app than the one that was currently configured.  In such a case, we won't have an appConfig object and this will just generate one for us.